src/main/webapp/WEB-INF/common/header.jspf

```diff
@@ -0,0 +1,27 @@
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/fmt" prefix="fmt"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<html>
+	<head>
+		<title>Todo List</title>
+		<link href="/webjars/bootstrap/5.2.0/css/bootstrap.min.css" rel="stylesheet">
+		<style>
```

Comment:
Move styles into a separate files

Reply:
a separate file for styles was created :)